src/utils/nodeController.js

```diff
@@ -6,24 +6,32 @@ const DockerCheck = require('../helpers/dockerCheck');
 const constants = require('./constants');
 const PREBUILT_CONFIGS = ['mainnet', 'testnet', 'previewnet', 'local'];
 const rootPath = process.cwd();
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
 
 module.exports = class NodeController {
-  static getNullOutput () {
-    if (constants.IS_WINDOWS) return 'null';
-    return '/dev/null';
+  static getConsoleOutput () {
+    const { verbose } = yargs(hideBin(process.argv)).argv;
+    if (verbose) {
+      if (constants.IS_WINDOWS) return '';
+      return '/dev/stdout';
```

Comment:
I just realized that this will almost only work on `OSX` so I will try it out with just `stdout` to see if it works both on `OSX` and `Linux`